wordpress/wp-content/plugins/sil-dictionary-webonary/include/dictionary-search.php

```diff
@@ -413,6 +416,25 @@ function replace_default_search_filter($input, $query=null)
 	return $input . PHP_EOL . getLimitSql();
 }
 
+/**
+ * Hook to override the default page length
+ *
+ * @param string $input
+ * @param WP_Query $query
+ *
+ * @return string
+ */
+function replace_default_search_limits($input, $query=null)
```

Comment:
I ended up using this hook because the paging was being calculated on 10 entries per page, regardless of what the SQL limit was.